package.json

```diff
@@ -18,12 +18,13 @@
     "graphql-tools": "^4.0.3"
   },
   "peerDependencies": {
-    "graphql": "^0.13.2"
+    "graphql": "^0.13.2 || ^14.0.0"
```

Comment:
any reason this can't work with the latest?

Reply:
Yes, it will break several things depending on the mix of modules you have. This module shouldn't even include graphql actually, because if multiple version exist they will explode.

Reply:
this is a common pattern I've seen in other graphql projects which is why I asked. we could also remove this line; graphql-tools has this same peerDependency and will complain for us if someone doesn't have `graphql` installed

https://github.com/apollographql/graphql-tools/blob/v4.0.3/package.json#L59